clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProviderSpec.groovy

```diff
@@ -64,6 +61,11 @@ class OpenstackClientProviderSpec extends Specification {
       String getTokenId() {
         null
       }
+
+      @Override
+      String getRegion(){
+        return ''
```

Comment:
Try and stay consistent with the quote type used, seems like you use `""` everywhere else